shared/src/main/scala/com/cibo/evilplot/plot/renderers/BoxRenderer.scala

```diff
@@ -30,22 +30,41 @@
 
 package com.cibo.evilplot.plot.renderers
 
-import com.cibo.evilplot.geometry.{Align, BorderRect, Drawable, Extent, Line, StrokeStyle}
+import com.cibo.evilplot.colors.{CategoricalColoring, Color}
+import com.cibo.evilplot.geometry.{Align, BorderRect, Drawable, Extent, Line, LineDash, Rect, StrokeStyle}
 import com.cibo.evilplot.numeric.BoxPlotSummaryStatistics
-import com.cibo.evilplot.plot.Plot
 import com.cibo.evilplot.plot.aesthetics.Theme
+import com.cibo.evilplot.plot.renderers.BoxRenderer.BoxRendererContext
+import com.cibo.evilplot.plot.{LegendContext, Plot}
 
-trait BoxRenderer extends PlotElementRenderer[BoxPlotSummaryStatistics] {
-  def render(plot: Plot, extent: Extent, summary: BoxPlotSummaryStatistics): Drawable
+trait BoxRenderer extends PlotElementRenderer[BoxRendererContext] {
+  def render(plot: Plot, extent: Extent, summary: BoxRendererContext): Drawable
+  def legendContext: LegendContext = LegendContext.empty
 }
 
 object BoxRenderer {
-  def default()(implicit theme: Theme): BoxRenderer = new BoxRenderer {
+  final case class BoxRendererContext(
+    summaryStatistics: BoxPlotSummaryStatistics,
+    index: Int
+  )
+
+  def default(
+    fillColor: Option[Color] = None,
+    strokeColor: Option[Color] = None,
+    lineDash: Option[LineDash] = None,
+    strokeWidth: Option[Double] = None
+  )(implicit theme: Theme): BoxRenderer = new BoxRenderer {
+    private val useFillColor = fillColor.getOrElse(theme.colors.fill)
+    private val useStrokeColor = strokeColor.getOrElse(theme.colors.path)
+    private val useLineDash = lineDash.getOrElse(theme.elements.lineDashStyle)
```

Comment:
useLineDash isn't used anywhere, could add it into the StrokeStyle though i.e. 
`StrokeStyle(useLineDash(Line(scale * topWhisker, useStrokeWidth)), useStrokeColor)`